README.md

```diff
@@ -1,3 +1,3 @@
-# Service Maker   [![Build Status](https://travis-ci.org/inetCatapult/service-maker.svg?branch=master)](https://travis-ci.org/inetCatapult/service-maker)
+# Service Maker   [![Build Status](https://travis-ci.org/trueblue2704/service-maker.svg?branch=task-2)](https://travis-ci.org/trueblue2704/service-maker)
```

Comment:
The build status badge should point to the `v1.0` branch of the `inetCatapult` service maker repo. This badge reflects the current status of the service maker build, so it needs to reflect the working branch (`v1.0`) of the main repo (`inetCatapult/service-maker`).